pippy/PipelineDriver.py

```diff
@@ -1433,6 +1435,17 @@ def forward(self, *args, **kwargs):
         if DEBUG:
             self._check_stages_cleanup()
 
```

Comment:
nit: (ignore if you need to land first)
When we have a chance, shall we add a comment here explaining the purpose of the code block below? Thanks!